### PR TITLE
Add current deployment config for the TrustyAI service

### PR DESCRIPTION
**Pull request by ruivieira**
@anishasthana: This is turrent OSD deployment for the trustyai-service, just for comparison.

### CODE REVIEW

**Comment by ruivieira** [comment]
@anishasthana can I ask you to have a look at we need to change, in order to merge this? Thank you!

File: explainability-service/manifests/opendatahub/default/trustyai-deployment.yaml [hunk 1]
```diff
@@ -40,7 +39,6 @@ metadata:
   name: trustyai-service
 spec:
   dockerImageRepository: quay.io/ruimvieira/trustyai-service
-
 ---
 apiVersion: apps.openshift.io/v1
 kind: DeploymentConfig
```
Comment (anishasthana) [inline explainability-service/manifests/opendatahub/default/trustyai-deployment.yaml]: Can you change this to kind `Deployment` and get rid of the deployment config specific things?

File: explainability-service/manifests/opendatahub/default/trustyai-deployment.yaml [hunk 2]
```diff
@@ -24,7 +24,6 @@ spec:
     app.kubernetes.io/name: trustyai-service
     app.kubernetes.io/version: 999-SNAPSHOT
   type: ClusterIP
-
 ---
 apiVersion: image.openshift.io/v1
```
Comment (anishasthana) [inline explainability-service/manifests/opendatahub/default/trustyai-deployment.yaml]: I would vote to delete this image stream

Comment (anishasthana) [reply explainability-service/manifests/opendatahub/default/trustyai-deployment.yaml]: Under the "containers" key in a deployment you can also specify an image, like so:
```
image: quay.io/ruimvieira/trustyai-service:<tag>
```

We should do that.
Related... should we move the image over to the trusyai org?

Comment (ruivieira) [reply explainability-service/manifests/opendatahub/default/trustyai-deployment.yaml]: @anishasthana Good point. We're moving it to https://quay.io/organization/trustyai

**Comment by ruivieira** [comment]
@anishasthana Removed `ImageStream`, moved env values to a `ConfigMap` and moved the image location to the new `trustyai` [quay.io org](https://quay.io/organization/trustyai).

(Just as a note, at the time of writing, the image is still not pushed to quay, but we plan to do it today).

**Comment by anishasthana** [comment]
Sounds good! Let me know once the image is pushed up. I can run through a test deployment to confirm. Related -- where is the dockerfile being used to build this image?

**Comment by RobGeada** [comment]
> Related -- where is the dockerfile being used to build this image?

@anishasthana The image is the same image as built by the live-builder

**Comment by RobGeada** [comment]
@ruivieira the `manifests/opendatahub/base/kustomization.yaml ` file needs to be:
```
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
commonLabels:
  app: trustyai
  app.kubernetes.io/part-of: trustyai
resources:
- ../default
- trustyai-configmap.yaml
```

**Comment by ruivieira** [comment]
Thanks @RobGeada, added.

**Review by RobGeada — APPROVED** [review]

**Review by tteofili — APPROVED** [review]

**Pull request merged**